infra/src/test/kotlin/fi/fta/geoviite/api/frameconverter/v1/TrackAddressToCoordinateIT.kt

```diff
@@ -223,15 +226,15 @@ class TrackAddressToCoordinateIT @Autowired constructor(
 
         val request = TestTrackAddressToCoordinateRequest(
             ratakilometri = 123,
-            ratametri = -1,
+            ratametri = -10001,
```

Comment:
Nykyisellään tämä on joo pakollinen näin sillä muuten testi ei mene läpi, mutta mielestäni jos/kun tiukennetaan `TrackMeter`:in validointia, niin tuo aiempi arvo olisi oikeastaan parempi

Reply:
Jep, samaa mieltä.